client/src/components/post/Comments.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+
+import { convertDate } from '../../utils/convertDate'
+import profilePicture from '../../assets/images/profile-picture.png'
+
+const Comments = (props) => {
+  const { comments } = props
```

Comment:
is it possible that `comments` is null or undefined?

Reply:
bc if it is, then your `returnComments` function will break

Reply:
I make sure comments comes back as an empty array if no comments